awx/main/tasks.py

```diff
@@ -3012,13 +3013,16 @@ def processor(self, resultfile):
 
     @property
     def receptor_params(self):
+        sdb.set_trace()
         if self.task.instance.is_container_group_task:
             spec_yaml = yaml.dump(self.pod_definition, explicit_start=True)
 
             receptor_params = {
                 "secret_kube_pod": spec_yaml,
             }
 
+            receptor_params["pod_pending_timeout"] = settings.AWX_CONTAINER_GROUP_POD_LAUNCH_TIMEOUT
```

Comment:
We might want to set a default, perhaps like this.  

```
getattr(settings, 'AWX_CONTAINER_GROUP_POD_LAUNCH_TIMEOUT', "5m")
```

Reply:
I have made the required changes